js/load.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+(function () {
+  var URL = 'https://javascript.pages.academy/code-and-magick/data';
+  var statusCode = {
+    OK: 200
+  };
+  var TIMEOUT_IN_MS = 10000;
```

Comment:
Хорошо бы четче пояснить, для чего именно TIMEOUT.

```suggestion
  var REQUEST_TIMEOUT_IN_MS = 10000;
```